src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/Snapshots/RestoreBuilder.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore
     /// </summary>
     internal static class RestoreBuilder
     {
-        public static readonly IVsTargetFrameworks2 EmptyTargetFrameworks = new TargetFrameworks(Enumerable.Empty<IVsTargetFrameworkInfo2>());
+        public static readonly IVsTargetFrameworks2 EmptyTargetFrameworks = new TargetFrameworks(Enumerable.Empty<IVsTargetFrameworkInfo3>());
```

Comment:
Should the field's type be changed too?

Reply:
No, they didn't version this type. It was a mistake that it was versioned from IVsTargetFrameworks -> IVsTargetFrameworks2 last time.